policyengine_us/variables/gov/states/ut/tax/income/ut_income_tax.py

```diff
@@ -7,5 +7,12 @@ class ut_income_tax(Variable):
     label = "Utah income tax"
     unit = USD
     definition_period = YEAR
-    adds = ["ut_income_tax_before_refundable_credits"]
     defined_for = StateCode.UT
+    reference = (
+        "https://tax.utah.gov/forms/current/tc-40.pdf#page=2"  # line 38
+    )
+
+    def formula(tax_unit, period, parameters):
+        income = tax_unit("ut_income_tax_before_refundable_credits", period)
+        credits = tax_unit("ut_refundable_credits", period)
+        return max_(income - credits, 0)
```

Comment:
This should be uncapped - refundable credits get paid out

Reply:
use adds/subtracts